app/controllers/api/stateless/payment_helper.rb

```diff
@@ -1,7 +1,7 @@
 # frozen_string_literal: true
 module PaymentHelper
   module Braintree
-    extend self
+    module_function
```

Comment:
To be honest I almost disabled this cop since I'm not sold on whatever the reasoning is behind defaulting to `module_function` instead of `extend self`. Also: http://idiosyncratic-ruby.com/8-self-improvement.html#which-one-to-use